mhw_library/test.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import xarray as xr
+import pytest
+
+def test_calc_duration():
+    # Create test data
+    ssta = xr.DataArray(np.random.rand(10, 10), dims=('time', 'space'))
+    labels = xr.DataArray(np.random.randint(0, 2, size=(10, 10)), dims=('time', 'space'))
+    merged_xarray = xr.Dataset({'SSTA': ssta, 'labels': labels})
+    
+    # Choose a specific object identifier to test
+    mhw_id = 1
+    
+    # Set a few timestamps to have the mhw_id label
+    labels[0, 0] = mhw_id
+    labels[1, 2] = mhw_id
+    labels[3, 5] = mhw_id
+    
+    # Call the function with the test data
+    result = calc_duration(merged_xarray, mhw_id)
```

Comment:
For the tests to work the functions should be imported from the library, or called with the library.module.function name